recipes/jaydebeapi/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "JayDeBeApi" %}
+{% set version = "1.1.1" %}
+{% set sha256 = "ba2dfa92c55e39476cea5a4b1a1750d94c8b3d166ed3c7f99601f19f744f2828" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
```

Comment:
I did not set `noarch: python` because I noticed that:
  - There are [two wheel files](https://pypi.org/project/JayDeBeApi/#files) in pypi, one for Python 2 and the other for Python 3. 
  - The code [checks](https://github.com/baztian/jaydebeapi/blob/e99a05d5a84e9aa37ff0bac00bd5591336f54402/jaydebeapi/__init__.py#L33) at runtime if it is running py2/3